core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Util.java

```diff
@@ -101,6 +106,17 @@ public static Map<String, String> authHeaders(String token) {
     }
   }
 
+  public static Map<String, String> authBasicHeader(String credential) {
```

Comment:
Shouldn't it be `basicAuth` rather than `authBasic`? The other methods that return a map also use a plural, `Headers` because there could be more than one.